MyoSimulatorForm/MyoSimulatorForm/HubCommunicator.cs

```diff
@@ -44,7 +44,7 @@ public enum Pose
             WAVE_IN,
             WAVE_OUT,
             FINGERS_SPREAD,
-            RESERVED1,
+            RESERVED1,      /* KEEP THIS! Must match the hub's pose enum */
             THUMB_TO_PINKY,
             UNKNOWN
         }
```

Comment:
What we can probably do is have the MyoSim check that the corresponding commands in this enum matches the Hub's enum when they both connect. If they don't match, issue some error warning and then close the sim.